pymdownx/snippets.py

```diff
@@ -199,10 +199,14 @@ def download(self, url):
 
             # We provide some basic protection against absurdly large files.
             # 32MB is chosen as an arbitrary upper limit. This can be raised if desired.
-            length = response.headers.get("content-length")
-            if length is None:
-                raise ValueError("Missing content-length header")
-            content_length = int(length)
+            content = None
+            if "content-length" not in response.headers:
+                # we have to read to know if we went over the max, but never more than url_max_size
```

Comment:
Please wrap `url_max_size` in backticks (`` ` ``) so that the spellchecker will ignore them.